internal/logger/logger_test.go

```diff
@@ -39,6 +45,114 @@ func BenchmarkLogger(b *testing.B) {
 	})
 }
 
+// mockKeyValues will return a slice of alternating keys and values of the
+// given length and the resulting JSON string.
+func mockKeyValuesB(b *testing.B, length int) KeyValues {
+	b.Helper()
```

Comment:
The call to `b.Helper()` here is unnecessary because there are no calls to `b` methods that need to know where in the call stack to find the code line number (e.g. `b.Error`). Remove `b *testing.B` from the function parameters and merge this function with `mockKeyValuesT`.